go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/processout/braintree-go
 
-go 1.21.1
+go 1.18
```

Comment:
That's the go version specified for Tessel9, where this library is used.
We won't need to upgrade Tessel9 version when importing this dependency.